README.md

```diff
@@ -114,14 +114,33 @@ var browzine = {
 ### BrowZine Primo Adapter Script
 Update the following code snippet with the `api` endpoint and `apiKey` values:
 ```
```

Comment:
FYI, most markdown renderers (including Github's) can use language specific syntax highlighting on code segments if you specify the language on the header like this:

````
```js
console.log('Highlighting!');
```
````

Renders to:
```js
console.log('Highlighting!');
```

It doesn't work for all languages, but if you happen to know the language code that works with your target it's a nice readability add.

Also, apparently you can escape triple backticks by using quadruple backticks above and below so you can display example markdown. TIL.

Reply:
TIL!